# 3.4 Discrete Probability Distributions #

## Objectives ##
- Apply probability and counting rules (including addition and multiplication rules) as pertaining to both continuous random variables and discrete random variables.
- Analyze an application in the disciplines business, social sciences, psychology, life sciences, health science, and education, and utilize the correct statistical processes to arrive at a solution.

## A Discrete Probability Distribution Function ##
A **discrete probability distribution function** has two characteristics:

1. Each probability is between zero and one, inclusive.
2. The sum of the probabilities is one.


***
### Example 4.1 ###
A child psychologist is interested in the number of times a newborn baby's crying wakes its mother after midnight. For a random sample of 50 mothers, the following information was obtained. Let $X$ = the number of times per week a newborn baby's crying wakes its mother after midnight. For this example, $x = 0, 1, 2, 3, 4, 5$.

$P(x)$ = probability that $X$ takes on a value $x$.

|x	|P(x)|
|--|--|
|0	|P(x = 0) =  $\dfrac{2}{50}$| 
|1	|P(x = 1) =  $\dfrac{11}{50}$|
|2	|P(x = 2) =  $\dfrac{23}{50}$ |
|3	|P(x = 3) =  $\dfrac{9}{50}$ |
|4	|P(x = 4) =  $\dfrac{4}{50}$ |
|5	|P(x = 5) =  $\dfrac{1}{50}$|

$X$ takes on the values 0, 1, 2, 3, 4, 5. This is a discrete PDF because:

1. Each $P(x)$ is between zero and one, inclusive.
2. The sum of the probabilities is one, that is,
$$ \frac{2}{50} + \frac{11}{50} + \frac{23}{50} + \frac{9}{50} + \frac{4}{50} + \frac{1}{50} = 1 $$

***
### Example 4.2 ###
Suppose Nancy has classes **three days** a week. She attends classes **three days a week 80%** of the time, **two days 15%** of the time, **one day 4%** of the time, and **no days 1%** of the time. Suppose one week is randomly selected.

1. What is $X$?
2. $X$ takes on what values?
3. Construct a probability distirbution table like the one in example 4.1. The table should have two columns labeled $x$ and $P(x)$. What does the $P(x)$ column sum up to?

#### Solution ####
##### Part 1 #####
$X$ = the number of days Nancy attends class.

##### Part 2 #####
$X$ takes on the values $x = 0, 1, 2, 3$.

##### Part 3 #####
|x	|P(x)|
|--|--|
|0	|0.01|
|1	|0.04|
|2	|0.15|
|3	|0.80|

If we add up the values in the $P(x)$ column, we get
$$0.01 + 0.04 + 0.15 + 0.80 = 1.00$$

***

## Expected Value ##
The **expected value** is often referred to as the **"long-term" average or mean**. This means that over the long term of doing an experiment over and over, you would **expect** this average.

You toss a coin and record the result. What is the probability that the result is heads? If you flip a coin two times, does probability tell you that these flips will result in one heads and one tail? You might toss a fair coin ten times and record nine heads. Recall that probability does not describe the short-term results of an experiment. It gives information about what can be expected in the long term. To demonstrate this, Karl Pearson once tossed a fair coin 24,000 times! He recorded the results of each toss, obtaining heads 12,012 times. **In his experiment, Pearson illustrated the Law of Large Numbers**.

**The Law of Large Numbers** states that, as the number of trials in a probability experiment increases, the difference between the theoretical probability of an event and the relative frequency approaches zero **(the theoretical probability and the relative frequency get closer and closer together)**. When evaluating the long-term results of statistical experiments, we often want to know the “average” outcome. This “long-term average” is known as the **mean** or **expected value** of the experiment and is denoted by the Greek letter $\mu$. In other words, after conducting many trials of an experiment, you would expect this average value.

The expected value of a discrete probability distribution function can be found using the formula
$$ \mu = \sum xP(x) $$
In words, we multiply each value $x$ by the probability $P(x)$ that the value will occur. Then we add up the products $xP(x)$.

***
### Example 4.3 ###
A men's soccer team plays soccer zero, one, or two days a week. The probability that they play zero days is 0.2, the probability that they play one day is 0.5, and the probability that they play two days is 0.3. Find the long-term average or expected value, $\mu$, of the number of days per week the men's soccer team plays soccer.

#### Solution ####

$X$ = the number of days the men's soccer team plays soccer per week. $X$ takes on the values 0, 1, 2 and has the probability distribution table below.

|x|P(x)|
|--|--|
|0	|0.2|	
|1	|0.5|	
|2	|0.3|

We will use R to find the expected value.

In [1]:
x <- c(0, 1, 2)
Px <- c(0.2, 0.5, 0.3)
xPx <- x * Px

mu <- sum(xPx)
print(mu)

[1] 1.1


The mean or expected value of the distribution is $\mu = 1.1$ games per week.

Note that the line
```R
xPx <- x * Px
```
multiplies the values in the list <code>x</code> by the corresponding probabilities in the list <code>Px</code>. The list of products is stored in <code>xPx</code>. For example, the first number in the list <code>xPx</code> is $0(0.2) = 0$, the second number is $1(0.5) = 0.5$, and the last number is $2(0.3) = 0.6$.

***
### Example 4.4 ###
Suppose you play a game of chance in which five numbers are chosen from 0, 1, 2, 3, 4, 5, 6, 7, 8, 9. A computer randomly selects five numbers from zero to nine with replacement. You pay \\$2 to play and could profit \\$100,000 if you match all five numbers in order (you get your \\$2 back plus \\$100,000). Over the long term, what is your **expected** profit of playing the game?

#### Solution ####
Let's first construct a probability distribution table. $X$ = the amount of money you profit from playing the game. The values that $X$ can take on are $x = -2$ and $x = 100,000$. (Note, since we are ultimately interested in the profit, the values are not 0, 1, 2, 3, 4, 5, 6, 7, 8, 9.)

Now let's look at the probability of winning the game. Let
- $N_1$ = the event you pick the right 1st number
- $N_2$ = the event you pick the right 2nd number
- $N_3$ = the event you pick the right 3rd number
- $N_4$ = the event you pick the right 4th number
- $N_5$ = the event you pick the right 5th number

The probability that you win is

$$ P(x = 100,000) = P(N_1\text{ AND }N_2\text{ AND }N_3\text{ AND }N_4\text{ AND }N_5) $$

Note that since the numbers are chosen *with replacement*, $N_1$, $N_2$, $N_3$, $N_4$, and $N_5$ are all independent events. Thus

$$\begin{align}
P(N_1\text{ AND }N_2\text{ AND }N_3\text{ AND }N_4\text{ AND }N_5) 
&= P(N_1)P(N_2)P(N_3)P(N_4)P(N_5) \\
&= \left(\frac{1}{10}\right)\left(\frac{1}{10}\right)\left(\frac{1}{10}\right)\left(\frac{1}{10}\right)\left(\frac{1}{10}\right)\\
&= \frac{1}{10^5} \\
&= 0.00001
\end{align}$$

So the probability that you win is $P(x = 100,000) = 0.00001$. Since there is a 100% chance that you either win or lose, the probability that you lose is

$$ P(x = -2) = 1.00 - 0.00001 = 0.99999. $$

The probability distribution table for this problem is

|  |	x|	P(x)|
|--|--|--|
|Lose|	–2|	0.99999|	
|Win|	100,000|	0.00001|

Now use R to calculate the expected value.


In [2]:
x <- c(-2, 100000)
Px <- c(0.99999, 0.00001)
xPx <- x * Px

mu <- sum(xPx)
print(mu)

[1] -0.99998


The expected value is $\mu = -\$0.99998$. This means if you played this game many times, you would lose an average of about \\$1 per game.
***

## Standard Deviation ##
The standard deviation, $\sigma$ of a discrete probability distribution function is

$$ \sigma = \sqrt{\sum\left[(x - \mu)^2 P(x)\right]}, $$

where $\mu$ is the expected value of the distribution.

When all outcomes in the probability distribution are equally likely, the expected value formula and the standard deviation formula above are equivalent to the mean and standard deviation formulas we learned earlier.

***
### Example 4.5 ###
Patients who have had an appendectomy must stay in the hospital for 1, 2, or 3 days. Patients stay in the hospital for 1 day 40% of the time, for 2 days 55% of the time, and for 3 days 5% of the time. 

1. How long should a patient **expect** to stay in the hospital? 
2. What is the standard deviation?

#### Solution ####
$X$ = the number of days the patient stays in the hosplital. The possible values of $X$ are $x = 1, 2, 3$. First, construct a probability distribution table.

| x | P(x) |
|--|--|
|1 | 0.40 |
|2 | 0.55 |
|3 | 0.05 |

##### Part 1 #####

In [4]:
x <- c(1, 2, 3)
Px <- c(0.40, 0.55, 0.05)

# Calculate the Mean
xPx <- x * Px
mu <- sum(xPx)
print(mu)

[1] 1.65


The expected value is $\mu = 1.65$. A patient should expect to spend, on average, 1.65 days in the hospital after surgery.

##### Part 2 #####

In [5]:
# Calculate the Standard Deviation
D <- (x - mu)^2 * Px
variance <- sum(D)
sigma <- sqrt(variance)
print(sigma)

[1] 0.5722762


So the standard deviation is $\sigma = 0.5722762$.

(Note that we use <code>D</code> to store the list we get from <code>(x - mu)^2 * Px</code> above just because we don't have a better name for it. We could also use <code>Q</code> or <code>K</code> or <code>Steve</code>.)

***
### Example 4.6 ###
A hospital researcher is interested in the number of times the average post-op patient will ring the nurse during a 12-hour shift. For a random sample of 50 patients, the following information was obtained.

|x	|P(x)|
|--|--|
|0	|P(x = 0) =  $\dfrac{4}{50}$ |
|1	|P(x = 1) =  $\dfrac{8}{50}$ |
|2	|P(x = 2) =  $\dfrac{16}{50}$ |
|3	|P(x = 3) =  $\dfrac{14}{50}$ |
|4	|P(x = 4) =  $\dfrac{6}{50}$ |
|5	|P(x = 5) =  $\dfrac{2}{50}$|

1. Find the mean.
2. Find the standard deviation.

#### Solution ####
##### Part 1 #####
Remember, for a probability distribution function, the mean and the expected value are the same thing.

In [7]:
x <- c(0, 1, 2, 3, 4, 5)
Px <- c(4/50, 8/50, 16/50, 14/50, 6/50, 2/50)

# Calculate the Mean
xPx <- x * Px
mu <- sum(xPx)
print(mu)

[1] 2.32


The mean or expected value is $\mu = 2.32$ times a patient will ring the nurse on average.

##### Part 2 #####

In [8]:
# Calculate the Standard Deviation
D <- (x - mu)^2 * Px
variance <- sum(D)
sigma <- sqrt(variance)
print(sigma)

[1] 1.223765


The standard deviation is $\sigma = 1.223765$.

***
## Attribution ##
Significant portions of this textbook were written by Barbara Illowsky and Susan Dean in their textbook *Introductory Statistics*, published by OpenStax.

Access for free at [https://openstax.org/books/introductory-statistics/pages/1-introduction](https://openstax.org/books/introductory-statistics/pages/1-introduction)